server/pulp/server/async/manage_workers.py

```diff
@@ -115,9 +115,9 @@ def _stop_workers():
         sys.exit(exit_code)
 
 
-if __name__ == "__main__":
+def main():
     """
-    This is the entry point method that becomes pulp-manage-workers.
+    This method is executed by the systemd unit file to manage the worker units.
```

Comment:
I know I originally used the word "method", but technically I think this is a function…